index.bs

```diff
@@ -1252,13 +1252,12 @@ the interaction with garbage collection.
 
 <h2 id="api-surface">JavaScript API Surface Concerns</h2>
 
-<h3 id="attributes-like-data">Attributes should behave like data properties</h3>
+<h3 id="accessors-like-data" oldids="attributes-like-data">Accessors should behave like data properties</h3>
 
-[[!WEBIDL]] attributes should act like simple JavaScript object properties.
+JavaScript accessors (specified as [[!WEBIDL]] attributes with separate getter and setter methods in Web APIs)
+should act like simple JavaScript object properties.
```

Comment:
```suggestion
JavaScript accessors 
should act like simple JavaScript object properties.
In [[!WEBIDL]], these are specified as attributes with separate getter and setter methods.
```

Reply:
The phrase "these are specified as attributes with separate getter and setter methods" implies that the getters and setters are explicit in the WebIDL, and they're generally implied.

Perhaps something like: "In [[!WEBIDL]] these are specified as attributes, which are implemented as getter and setter methods."

Reply:
Web IDL attributes are used for both (writable) data properties as well as accessors. My understanding of what this sentence was saying is that in WebIDL, to specify an accessor, you specify an attribute with separate getter and setter.